### PR TITLE
[WFLY-17111] WildFly documentation fixes

### PR DESCRIPTION
**Pull request by boris-unckel**
Fixes https://issues.redhat.com/browse/WFLY-17111

### CODE REVIEW

**Comment by boris-unckel** [comment]
Two open issues:
* [INFO] asciidoctor: ERROR: Getting_Started_Developing_Applications_Guide.adoc: line 32: include uri not readable: https://raw.githubusercontent.com/wildfly/quickstart/27.x/guide/Introduction.asciidoc (and others) ==> because the 27.x branch does not exist yet.
* [INFO] asciidoctor: INFO: possible invalid reference: sts-implementation-class ==> I did not find the corresponding chapter

File: docs/src/main/asciidoc/_developer-guide/Jakarta_Persistence_Reference_Guide.adoc [hunk 1]
```diff
@@ -925,7 +925,7 @@ public class ExampleSFSB {
 
 == Tips for using the latest Hibernate ORM
 
-http://docs.jboss.org/hibernate/stable/orm/topical/html_single/wildfly/Wildfly.html[Read about using Hibernate ORM feature packs on WildFly].
+https://docs.jboss.org/hibernate/orm/5.4/topical/html_single/wildfly/Wildfly.html[Read about using Hibernate ORM feature packs on WildFly].
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_developer-guide/Jakarta_Persistence_Reference_Guide.adoc:928]: I believe L926-928 should just be removed.  The linked doc is about the pre-Galleon feature packs, and WildFly hasn't provided those for quite a long time now, so what the doc says no longer works. I assume that is why Hibernate removed that doc from their 6.x docs.

@scottmarlow FYI.

**Review by bstansberry — CHANGES_REQUESTED** [review]
Thanks @boris-unckel. I have one small suggestion re ORM' otherwise LGTM.

File: docs/src/main/asciidoc/_developer-guide/Jakarta_Persistence_Reference_Guide.adoc (hunk 1 above)
Comment (boris-unckel) [reply docs/src/main/asciidoc/_developer-guide/Jakarta_Persistence_Reference_Guide.adoc:928]: Thanks for your feedback, I'm preparing a fix and remove the link.

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @boris-unckel

File: docs/src/main/asciidoc/_developer-guide/Jakarta_Persistence_Reference_Guide.adoc (hunk 1 above)
Comment (scottmarlow) [reply docs/src/main/asciidoc/_developer-guide/Jakarta_Persistence_Reference_Guide.adoc]: Thanks Boris for noticing the Hibernate ORM support for adding new ORM releases on WildFly.  I agree with Brian as that was later removed from the Hibernate ORM project.

**Pull request merged**